govc/USAGE.md

```diff
@@ -4291,6 +4336,32 @@ Options:
   -cluster=              Cluster [GOVC_CLUSTER]
 ```
 
+## namespace.ls
+
+```
+Usage: govc namespace.ls [OPTIONS]
+
+Displays the list of vSphere Namespaces. 
+
+Examples:
+  govc namespace.ls
+
+Options:
+```
+
+## namespace.rm
+
+```
+Usage: govc namespace.rm [OPTIONS] NAME
+
+Deletes a namespace. 
```

Comment:
I think we should be consistent: we call it `vSphere Namespace` or `Supervisor namespace` and here just `namespace`.

Maybe use `vSphere Namespace` everywhere?

Reply:
Agreed, I replaced `namespace` with `vSphere Namespace` in the docs for this and other commands